config.json

```diff
@@ -63,6 +63,13 @@
 				"did:ethr:0x1e:0x3b0BC51Ab9De1e5B7B6E34E5b960285805C41736",
 				"did:ethr:0x1e:0x03fdd57adec3d438ea237fe46b33ee1e016eda6b585c3e27ea66686c2ea5358479"
 			]
+		}, {
+			"pattern": "^(did:eosio:.+)$",
+			"url": "http://eosio-driver:8081/",
```

Comment:
The port should be changed to `8080` because which is the container port given in the docker-compose file.

Reply:
fixed